Aztec/Classes/Converters/HTMLNodeToNSAttributedString.swift

```diff
@@ -113,13 +113,12 @@ class HMTLNodeToNSAttributedString: SafeConverter {
         let childAttributes = attributes(forNode: node, inheritingAttributes: inheritedAttributes)
 
         for child in node.children {
-            let childContent = convert(child, inheritingAttributes: childAttributes)
-
+            let childContent = NSMutableAttributedString(attributedString:convert(child, inheritingAttributes: childAttributes))
```

Comment:
You are right, I changed it for the previous version of code, and the forgot to change it back when moved the code away from this method. I will clear this out.